### PR TITLE
Andy - Hito 2

### PR DESCRIPTION
**Pull request by andyOlarte514**
- https://trello.com/c/i9WWBi2l/50-fe-hito-2-andy-1-vista-de-perfil-de-usuario
 - https://trello.com/c/dNYtHrAi/47-fe-hito-2-andy-1-bug-tipo-de-proyecto-no-está-filtrando-en-la-búsqueda-de-productos
 - https://trello.com/c/cxCHho5F/51-fe-hito-2-andy-3-cada-vez-que-se-seleccione-un-filtro-los-filtros-deben-resetear-sus-opciones-con-el-nuevo-filtro-incluído

### CODE REVIEW

File: src/assets/styles/components/input-range.scss [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        border: solid 2px #46c0ae;
+        border-radius: 50%;
+    }
+
+    &::-ms-thumb {
+        background-color: #ffffff;
+        width: 17px;
+        height: 17px;
+        border: solid 2px #46c0ae;
+        border-radius: 50%;
```
Comment (Nosoyleon) [inline src/assets/styles/components/input-range.scss:42]: Maybe for future PR's we should add a partial with colors  variables for avoiding repeating across the file.

File: src/components/buttons/Button.jsx [hunk 2]
```diff
@@ -15,27 +15,20 @@ import {
...
+    case 'secondary':
+      return <SecondaryButton {...buttonProps} />;
+    case 'gray':
+      return <GrayButton {...buttonProps} />;
+    case 'cancel':
+      return <CancelButton {...buttonProps} />;
+    case 'cancel-secondary':
+      return <CancelSecondaryButton {...buttonProps} />;
+    default:
+      return <DefaultButton {...buttonProps} />;
```
Comment (Nosoyleon) [inline src/components/buttons/Button.jsx:30]: What do you think about, making the variant options a enum. And then we can have a hoc that can return `cancelButton[varian]` in that way we could avoid the complete switch. But it is not a blocker we can discuss it in future PR's

File: src/components/navbar/navbar-profile/NavProfile.styles.jsx [hunk 3]
```diff
@@ -53,7 +54,18 @@ export const OptionsContent = styled.section`
...
+export const ProfilePictureContainer = styled.div`
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  overflow: hidden;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 30px;
 `;
```
Comment (Nosoyleon) [inline src/components/navbar/navbar-profile/NavProfile.styles.jsx:71]: Add the missing line empty line at the end.

File: src/containers/profile-change-picture/ProfileChangePicture.jsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+                    <Button variant="primary" width="113px" onClick={handleSaveProfilePicture} disabled={!avatar?.name}>
+						Guardar
+					</Button>
+                </Footer>
+            </Root>
+        </ModalLayout>
+    );
+};
+
+export default ProfileChangePicture;
```
Comment (Nosoyleon) [inline src/containers/profile-change-picture/ProfileChangePicture.jsx:102]: EOF

File: src/containers/profile-header/ProfileHeader.styles.jsx [hunk 5]
```diff
@@ -188,4 +189,14 @@ export const TextValue = styled.div`
...
+
+export const ButtonCreateContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  padding: 0 81px;
+`;
+
+export const ProfileNameInputContainer = styled.div`
+  max-width: 160px;
+`;
```
Comment (Nosoyleon) [inline src/containers/profile-header/ProfileHeader.styles.jsx:202]: EOF

File: src/views/Products.styles.jsx [hunk 6]
```diff
@@ -5,8 +5,12 @@ export const Container = styled.section`
...
+export const PaddingContainer = styled.div`
+  padding: 0 81px;
+`;
+
 export const Separator = styled.div`
-  height: 2px;
+  height: 1px;
   width: 100%;
   background-color: ${COLOR_LIGHTERGREY};
 `;
```
Comment (Nosoyleon) [inline src/views/Products.styles.jsx:16]: EOF

**Review by Nosoyleon — CHANGES_REQUESTED** [review]

File: src/assets/styles/components/input-range.scss (hunk 1 above)
Comment (andyOlarte514) [reply src/assets/styles/components/input-range.scss:42]: I agree, let's talk to Sergio and we will do it as a refactor for future Hitos

File: src/components/buttons/Button.jsx (hunk 2 above)
Comment (andyOlarte514) [reply src/components/buttons/Button.jsx:30]: same, I agree, let's talk to Sergio and we will do it as a refactor for future Hitos

Comment (Sophia-Sergio) [reply src/components/buttons/Button.jsx:30]: how much effort it takes in hours? if it is less than 1 hour please do it and discount time from the next milestone (hito 😬 )

File: src/assets/styles/components/input-range.scss (hunk 1 above)
Comment (Sophia-Sergio) [reply src/assets/styles/components/input-range.scss:42]: how much effort it takes in hours? if it is less than 1 hour please do it and discount time from the next milestone (hito 😬 )

**Review by Nosoyleon — APPROVED** [review]

**Pull request merged**